presto-orc/src/test/java/io/prestosql/orc/metadata/statistics/TestDoubleStatisticsBuilder.java

```diff
@@ -34,7 +34,7 @@
 
     public TestDoubleStatisticsBuilder()
     {
-        super(DOUBLE, DoubleStatisticsBuilder::new, DoubleStatisticsBuilder::addValue);
+        super(DOUBLE, () -> new DoubleStatisticsBuilder(new NoOpBloomFilterBuilder()), DoubleStatisticsBuilder::addValue);
```

Comment:
I'd like to see the bloom filter actually tested in these.  For example the `assertValues` in the parent class could verify the expected values are matched by the bloom filter.